RadarSDK/RadarLocationManager.m

```diff
@@ -204,13 +249,18 @@ - (void)startTrackingWithOptions:(RadarTrackingOptions *)trackingOptions {
 
 - (void)stopTracking {
     [RadarSettings setTracking:NO];
+
     
     RadarSdkConfiguration *sdkConfiguration = [RadarSettings sdkConfiguration];
     if (sdkConfiguration.extendFlushReplays) {
         [[RadarLogger sharedInstance] logWithLevel:RadarLogLevelInfo type:RadarLogTypeSDKCall message:@"Flushing replays from stopTracking()"];
         [[RadarReplayBuffer sharedInstance] flushReplaysWithCompletionHandler:nil completionHandler:nil];
     }
 
+    if (sdkConfiguration.useLocationMetadata) {
```

Comment:
Do we also need to call this inside `updateTracking`? I don't think this method gets called by `updateTracking`